openshift/Jenkinsfile.ocp4

```diff
@@ -50,14 +50,15 @@ pipeline {
                     sh "PROJ_TARGET=${projProd} PROJ_TOOLS=e1e498-tools IMAGE_REGISTRY=image-registry.openshift-image-registry.svc:5000 POSTGRES_WRITE_HOST=patroni-wps-prod-leader POSTGRES_READ_HOST=patroni-wps-prod-replica POSTGRES_DATABASE=wps POSTGRES_USER=wps ./openshift/scripts/oc_provision_ec_hrdps_cronjob.sh prod apply"
                     sh "PROJ_TARGET=${projProd} PROJ_TOOLS=e1e498-tools IMAGE_REGISTRY=image-registry.openshift-image-registry.svc:5000 POSTGRES_WRITE_HOST=patroni-wps-prod-leader POSTGRES_READ_HOST=patroni-wps-prod-replica POSTGRES_DATABASE=wps POSTGRES_USER=wps ./openshift/scripts/oc_provision_ec_rdps_cronjob.sh prod apply"
                     // BC FireWeather cronjobs
-                    sh "PROJ_TARGET=${projProd} PROJ_TOOLS=e1e498-tools IMAGE_REGISTRY=image-registry.openshift-image-registry.svc:5000 POSTGRES_WRITE_HOST=patroni-wps-prod-leader POSTGRES_READ_HOST=patroni-wps-prod-replica POSTGRES_DATABASE=wps POSTGRES_USER=wps SCHEDULE=\"30 8,16 * * *\" ./openshift/scripts/oc_provision_bcfw_p1_forecasts_cronjob.sh prod apply"
-                    sh "PROJ_TARGET=${projProd} PROJ_TOOLS=e1e498-tools IMAGE_REGISTRY=image-registry.openshift-image-registry.svc:5000 POSTGRES_WRITE_HOST=patroni-wps-prod-leader POSTGRES_READ_HOST=patroni-wps-prod-replica POSTGRES_DATABASE=wps POSTGRES_USER=wps SCHEDULE=\"15 * * * *\" ./openshift/scripts/oc_provision_bcfw_p1_hourly_actuals_cronjob.sh prod apply"
+                    // TODO: Change cron to: 30 8,16 * * * (was set 2 minutes early to avoid clash with pathfinder instance)
```

Comment:
are this and the other still just todos?

Reply:
Yes. We need pathfinder and silver to run at different times, otherwise we get 500 errors from the legacy system.